BeagleBone/Black/input2.js

```diff
@@ -1,31 +1,41 @@
 #!/usr/bin/env node
+////////////////////////////////////////
```

Comment:
For PocketBeagle, I did a single Fritzing diagram to show all demo hookups. Is it useful? Should each directory have a diagram in the README?

Reply:
I can see the diagram being helpful to beginners.  I have a Fritzing diagram for the Black.  Is there one for the AI?

Reply:
No, there isn't an AI version yet. I don't have any immediate plans to make one---so maybe you can? ;-)

Reply:
Let me look into it.  It might be a fun project.

Reply:
Well, maybe not so fun.  I was hoping to edit a Black diagram, but couldn't figure it out.